js/index.js

```diff
@@ -8,35 +8,156 @@ const siteContent = {
+
+const ctaText = document.querySelector(".cta-text");
+const ctaImg = document.querySelector("#cta-img");
+const textContents = document.querySelectorAll(".text-content");
+const middleImg = document.querySelector("#middle-img");
+const contact = document.querySelector(".contact");
+
+const br = document.createElement("br");
+
+navItems[0].textContent = siteContent.nav["nav-item-1"];
+navItems[1].textContent = siteContent.nav["nav-item-2"];
+navItems[2].textContent = siteContent.nav["nav-item-3"];
+navItems[3].textContent = siteContent.nav["nav-item-4"];
+navItems[4].textContent = siteContent.nav["nav-item-5"];
+navItems[5].textContent = siteContent.nav["nav-item-6"];
+
+const aTag1 = document.createElement("a");
+nav.prepend(aTag1);
+aTag1.textContent = "NavItem 1";
+aTag1.href = "#";
+
+navItems = document.querySelectorAll("nav > a");
+navItems.forEach(nav => (nav.style.color = "green"));
+
+ctaText.children[0].textContent = siteContent.cta.h1;
+ctaText.children[1].textContent = siteContent.cta.button;
+
+const headline = ctaText.children[0];
+const button = ctaText.children[1];
+
+button.addEventListener("click", () => {
+  const colors = ["green", "yellow", "red", "blue", "lightBlue", "orange"];
+  headline.style.color = colors[Math.round(Math.random() * colors.length)];
+});
+
+// ctaImg.src = siteContent.cta["img-src"];
+// middleImg.src = siteContent["main-content"]["middle-img-src"];
+
+// textContents[0].children[0].textContent =
+//   siteContent["main-content"]["features-h4"];
+
+textContents[0].children[1].textContent =
+  siteContent["main-content"]["features-content"];
+
+// textContents[1].children[0].textContent =
+//   siteContent["main-content"]["about-h4"];
+
+textContents[1].children[1].textContent =
+  siteContent["main-content"]["about-content"];
+
+// textContents[2].children[0].textContent =
+//   siteContent["main-content"]["services-h4"];
+
+textContents[2].children[1].textContent =
+  siteContent["main-content"]["services-content"];
+
+// textContents[3].children[0].textContent =
+//   siteContent["main-content"]["product-h4"];
+
+textContents[3].children[1].textContent =
+  siteContent["main-content"]["product-content"];
+
+// textContents[4].children[0].textContent =
+//   siteContent["main-content"]["vision-h4"];
+
+textContents[4].children[1].textContent =
+  siteContent["main-content"]["vision-content"];
+
+contact.children[0].textContent = siteContent.contact["contact-h4"];
+contact.children[1].textContent = siteContent.contact.address;
+contact.children[2].textContent = siteContent.contact.phone;
+contact.children[3].textContent = siteContent.contact.email;
+
+const footer = (document.querySelector("footer").children[0].textContent =
+  siteContent.footer.copyright);
+
+let array = [];
+
+// Loop through each key in siteContent["main-content"]. If the key includes "h4" in it, then we push it into an array
+for (let i = 0; i < Object.keys(siteContent["main-content"]).length; i++) {
```

Comment:
This was a lot of fun to work on together. I completely forgot we could use Object.keys to do this.